test/Github/Tests/Mock/PaginatedResponse.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Github\Tests\Mock;
+
+use GuzzleHttp\Psr7\Response;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class PaginatedResponse extends Response
+{
+    protected $loopCount;
+
+    protected $content;
+
+    public function __construct($loopCount, array $content = array())
```

Comment:
as we have shifted to 5.5 version I would like to see square brackets declarations for arrays in the new code